python/lsst/cp/verify/verifyBias.py

```diff
@@ -150,11 +152,16 @@ def verify(self, exposure, statisticsDict):
             # consistently and significantly, then the assumptions
             # used in that test may be incorrect, and the nominal read
             # noise may need recalculation.  Only perform this check
-            # if the metadataStats contain the required entry.
-            overscanReadNoise = metadataStats['READ_NOISE'][ampName]
+            # if the metadataStats contain the required entry.  This
+            # is in ADU (the serial overscan is measured prior to gain
+            # normalization), so we need to convert to electrons here.
+            gain = gains[ampName]
+            overscanReadNoise = gain * metadataStats['READ_NOISE'][ampName]
```

Comment:
This is not correct.  The readnoise units are electrons https://github.com/lsst/ip_isr/blob/0d33ba980625e815f92b36fa222f8240fc568f41/python/lsst/ip/isr/isrTaskLSST.py#L1757C9-L1757C58 and you can look at the header to confirm this.  But now I'm disappointed that https://github.com/lsst/ip_isr/blob/0d33ba980625e815f92b36fa222f8240fc568f41/python/lsst/ip/isr/isrFunctions.py#L1616 doesn't make this clear.

Reply:
Okay, this code is correct but confusing because of the units.  Is it possible to ...
* rename the metadata stats (in dark and bias verification) to `READ_NOISE_ADU`
* Maybe call this `overscanReadNoiseElectron` (but if on the right it says `READ_NOISE_ADU` I think that would be obvious)
* Add a comment above on `readNoises` that this is returning the read noise in electrons.
* And in the dense `DMTN-101 Test 4.2: Mean is 0.0 within noise.` comment above say that "the nominal detector read noise is in electron units"

Reply:
Agree on the first, left the second alone, as I think the changed key name makes that clear (and the preceding comment notes we're converting to electrons.  
Added comment above the gainDict/readnoiseDict noting that these are the PTC values used in constructing the variance plane (and that RN is in electrons).  
I've expanded the Test 4.3/5.3 comments a bit to note that this test runs against the PTC values.
I've clarified the Test 4.2/5.2 is comparing the mean to the noise measured on the image (which is in e-).
Hopefully clearer now (and perhaps in the future I will not name four different things "noise").